src/config/Blocks.jsx

```diff
@@ -300,6 +301,9 @@ const blocksConfig = {
     view: ViewVideoBlock,
     edit: EditVideoBlock,
     schema: BlockSettingsSchema,
+    blockSchema: VideoBlockSchema,
+    extensions: {},
```

Comment:
This is not needed.

Reply:
nor variations.

Reply:
@danielamormocea could you please take care?